ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -73,158 +77,289 @@ export default function OnboardingMetametrics() {
+              {t('onboardingMetametricsNeverCollect', [
+                <Typography
+                  variant={TypographyVariant.span}
+                  key="never"
+                  fontWeight={FONT_WEIGHT.BOLD}
+                  marginTop={0}
+                >
+                  {t('onboardingMetametricsNeverCollectEmphasis')}
+                </Typography>,
+              ])}
+            </Box>
+          </li>
+          <li>
+            <Box>
+              <Icon
+                marginInlineEnd={2}
+                name={IconName.Check}
+                size={IconSize.Sm}
+                color={IconColor.successDefault}
+              />
+              {t('onboardingMetametricsNeverCollectIP', [
+                <Typography
+                  variant={TypographyVariant.span}
+                  key="never-collect"
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  {t('onboardingMetametricsNeverCollectIPEmphasis')}
+                </Typography>,
+              ])}
+            </Box>
+          </li>
+          <li>
+            <Box>
+              <Icon
+                marginInlineEnd={2}
+                name={IconName.Check}
+                size={IconSize.Sm}
+                color={IconColor.successDefault}
+              />
+              {t('onboardingMetametricsNeverSellData', [
+                <Typography
+                  variant={TypographyVariant.span}
+                  key="never-sell"
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  {t('onboardingMetametricsNeverSellDataEmphasis')}
+                </Typography>,
+              ])}
+            </Box>{' '}
+          </li>
+        </ul>
+        <Typography
+          color={TextColor.textAlternative}
+          align={TEXT_ALIGN.LEFT}
+          variant={TypographyVariant.H6}
+          className="onboarding-metametrics__terms"
         >
-          {t('onboardingMetametricsDisagree')}
-        </Button>
+          {t('onboardingMetametricsInfuraTerms', [
+            <a
+              href="https://metamask.io/privacy.html"
+              target="_blank"
+              rel="noopener noreferrer"
+              key="privacy-link"
+            >
+              {t('onboardingMetametricsInfuraTermsPolicy')}
+            </a>,
+          ])}
+        </Typography>
+
+        <div className="onboarding-metametrics__buttons">
+          <Button
+            data-testid="metametrics-i-agree"
+            type="primary"
+            large
+            onClick={onConfirm}
+          >
+            {t('onboardingMetametricsAgree')}
+          </Button>
+          <Button
+            data-testid="metametrics-no-thanks"
+            type="secondary"
+            large
+            onClick={onCancel}
+          >
+            {t('onboardingMetametricsDisagree')}
+          </Button>
+        </div>
       </div>
-    </div>
-  );
+    );
+  };
+
+  return currentDate >= newPrivacyPolicyDate
```

Comment:
Nit: can we add a `TODO` so that we don't forget about the possibility of removing this and making it simple again in the future?

```suggestion
  // TODO: The legacy onboarding can be completely removed after 2024-06-04
  return currentDate >= newPrivacyPolicyDate
```

Reply:
We will probably reuse this in the future, once the privacy policy gets updated once again